mobile/src/main/java/ch/epfl/sdp/mobile/infrastructure/persistence/store/CollectionReference.kt

```diff
@@ -8,6 +8,13 @@ import kotlinx.coroutines.flow.Flow
  */
 interface CollectionReference : Query {
 
+  /**
+   * Accesses (creates) a document with no specified path (generates one automatically).
```

Comment:
I'd rather say something along the lines of "Creates a document with a unique identifier."